service_user.go

```diff
@@ -27,14 +31,20 @@ type (
 	// CreateServiceUserRequest are the parameters required to create a
 	// ServiceUser.
 	CreateServiceUserRequest struct {
-		Username string `json:"username"`
+		Username           string   `json:"username"`
+		RedisACLCategories []string `json:"redis_acl_categories,omitempty"`
+		RedisACLCommands   []string `json:"redis_acl_commands,omitempty"`
+		RedisACLKeys       []string `json:"redis_acl_keys,omitempty"`
 	}
 
 	// ModifyServiceUserRequest params required to modify a ServiceUser
 	ModifyServiceUserRequest struct {
-		Operation      *string `json:"operation"`
-		Authentication *string `json:"authentication,omitempty"`
-		NewPassword    string  `json:"new_password"`
+		Operation          *string  `json:"operation"`
+		Authentication     *string  `json:"authentication,omitempty"`
+		NewPassword        string   `json:"new_password"`
+		RedisACLCategories []string `json:"redis_acl_categories,omitempty"`
+		RedisACLCommands   []string `json:"redis_acl_commands,omitempty"`
+		RedisACLKeys       []string `json:"redis_acl_keys,omitempty"`
```

Comment:
As far as I can see updating these isn't currently possible. Probably it should be.